i18n/en.json

```diff
@@ -177,6 +177,18 @@
     },
     "hems": {
       "description": "Connect evcc to another home energy management system.",
+      "shm": {
+        "cardTitle": "Sunny Home Manager",
+        "descriptionDeviceid": "TODO",
+        "descriptionVendorid": "TODO",
```

Comment:
```suggestion
        "descriptionVendorid": "This allows you to specify a custom vendor-specific part of the SEMP-ID, otherwise the built-in identifier will be used.",
```